### PR TITLE
TorchVision

### PR DESCRIPTION
**Pull request by NiklasGustafsson**
This completes the torchvision.transforms module.
There are a number of tensor class instance functions that are duplicated in 'torch'. This work uncovered a few more of them, but there is no guarantee that the work is completed.

### CODE REVIEW

**Comment by NiklasGustafsson** [comment]
@dsyme -- any comments? I'll merge later today unless you object.

File: src/Examples/ImageTransforms.cs [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+                );
+
+            var second = transform.forward(first);
+
+            for (; n < second.shape[0]; n++) {
+
+                var image = second[n]; // CxHxW
+                var channels = image.shape[0];
+
+                using (var stream = File.OpenWrite(@"D:\repos\niklasgustafsson\TorchSharp\output-" + n + ".png")) {
```
Comment (dsyme) [inline src/Examples/ImageTransforms.cs:46]: There's a hardwired path here

File: src/Examples/ImageTransforms.cs [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+using System.Runtime.InteropServices;
+
+namespace TorchSharp.Examples
+{
+    public class ImageTransforms
+    {
+        internal static void Main(string[] args)
+        {
+            var images = new string[] {
+                @"D:\repos\mono\SkiaSharp\samples\Gallery\Shared\Media\baboon.png",
```
Comment (dsyme) [inline src/Examples/ImageTransforms.cs:19]: Do we mind about hardwired paths?

File: src/Examples/ImageTransforms.cs [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+            second = torchvision.transforms.functional.convert_image_dtype(second, dtype: ScalarType.Byte);
+            second = torchvision.transforms.functional.equalize(second);
+            second = torchvision.transforms.functional.resize(second, 256, 256);
+
+            for (n = 0; n < second.shape[0]; n++) {
+
+                var image = second[n]; // CxHxW
+                var channels = image.shape[0];
+
+                using (var stream = File.OpenWrite(@"D:\repos\niklasgustafsson\TorchSharp\output-" + (n + first.shape[0]) + ".png")) {
```
Comment (dsyme) [inline src/Examples/ImageTransforms.cs:65]: Another hardwired path

Comment (NiklasGustafsson) [reply src/Examples/ImageTransforms.cs:65]: That's not okay. Will fix.

**Review by dsyme — APPROVED** [review]

**Comment by dsyme** [comment]
Fantastic work

**Comment by dsyme** [comment]
Left a few comments, looks great

**Pull request merged**